DataFormats/BTauReco/interface/ParticleTransformerAK4Features.h

```diff
@@ -0,0 +1,27 @@
+#ifndef DataFormats_BTauReco_ParticleTransformerAK4Features_h
+#define DataFormats_BTauReco_ParticleTransformerAK4Features_h
+
+//#include "DataFormats/BTauReco/interface/JetFeatures.h"
```

Comment:
Is this commented out code needed here? If not, please remove

Reply:
Will remove the commented-out lines, thanks for pointing this out.
A bit more context: These global jet features currently only play a role for phase 2 developments, and have been used in the current DeepJet tagger, which is why these lines were included in the first place. As our current transformer model does not use them, I agree that these lines should be removed for now.